client/verta/tests/unit_tests/deployment/test_deployed_model.py

```diff
@@ -10,8 +10,6 @@
 
```

Comment:
Stop! I'd recommend going to https://github.com/VertaAI/modeldb/pull/4103/commits and reviewing the non-"Apply autoformatting" commits.